main/calculations.py

```diff
@@ -0,0 +1,147 @@
+import requests
+import json
+from datetime import datetime
+from constance import config
+from proxy_requests.proxy_requests import ProxyRequests
+
+from django.core.mail import send_mail
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.contrib import messages
+
+from .models import SiteToCheck
+
+
+def send_email(message, email):
+    subject = 'Errors on my pages'
+    message = message
+    from_email = settings.EMAIL_ADDRESS
+    to_email = email
+    send_mail(
+        subject,
+        message,
+        from_email,
+        [to_email],
+        fail_silently=False,
+    )
+
+
+def my_cron_job():
+    users = User.objects.all()
+    for user in users:
+        sites = SiteToCheck.objects.filter(user=user)
+        output = ''
+        for site in sites:
+            if 'bad_data' in SiteDownChecker(site, user).status():
+                output += f'{site} - ERROR\n'
+            elif site.last_status != 200:
+                output += f'{site} - last status: {site.last_status}'
+        if len(output) > 0:
+            send_email(output, user.email)
+
+
+def modify_email(request):
```

Comment:
To się powinno nazywać update_user_email. Wtedy na pierwszy rzut oka widać co ta funkcja robi.